buddyboss/buddyboss-platform
# Group Navigation changes after video tab added

### All Submissions:






### Changes proposed in this Pull Request:
Proof screenshot : https://www.awesomescreenshot.com/image/10652469?key=00d023fa715f80cde79672df006c9614



Fixes #2581 

### How to test the changes in this Pull Request:

1. Go to Group Navigation
2. Check different menu items and check the same order on group page  
3. both these place has different order


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
